conu/backend/docker/image.py

```diff
@@ -283,6 +284,11 @@ def callback():
             return subprocess.Popen(run_command_instance.build(), **popen_params)
         container_id, popen_instance = self._run_container(run_command_instance, callback)
 
+        actual_name = self.d.inspect_container(container_id)['Name'][1:]
+        if container_name and container_name != actual_name:
+            raise ConuException("Container name  is not set to desired value")
```

Comment:
nit: could we display container_name and actual_name here? there is also double space in the message